src/js/apps/patients/sidebar/filters-sidebar_app.js

```diff
@@ -39,10 +55,49 @@ export default App.extend({
 
     this.showChildView('customFilters', customFiltersView);
   },
+  showStatesFiltersView() {
+    const statesFiltersView = new StatesFiltersView({
+      collection: this.defaultStates,
+      state: this.getState(),
+    });
+
+    this.showChildView('statesFilters', statesFiltersView);
+  },
   onClearFilters() {
-    this.getState().clear();
+    const defaultStates = this.defaultStates.map('id').reduce((states, id) => {
+      states[id] = true;
+      return states;
+    }, {});
+
+    this.getState().clear().set({ states: defaultStates });
```

Comment:
When the state is cleared via the `Clear Filters` button, set the default states to be selected.